draft-ietf-tls-tls13.md

```diff
@@ -5465,8 +5465,8 @@ endpoints potentially supporting different versions of TLS.
 
 TLS 1.x and SSL 3.0 use compatible ClientHello messages. Servers can also handle
 clients trying to use future versions of TLS as long as the ClientHello format
-remains compatible and the client supports the highest protocol version available
-in the server.
+remains compatible and and there is at lest one protocol version that both the
+client and the server support.
```

Comment:
I might word it as "there is at least one protocol version supported by both the client and the server", but that's editorial.
Overall this seems an okay direction to go in.

Reply:
@kaduk agree, your version is easier to read.